rust/operator/src/kafka_controller.rs

```diff
@@ -934,9 +936,13 @@ fn build_broker_rolegroup_statefulset(
     }
 
     let mut pod_template = pod_builder.build_template();
+    pod_template.merge_from(role.config.pod_overrides.clone());
+    pod_template.merge_from(rolegroup.config.pod_overrides.clone());
```

Comment:
Can we please move this down, so that we get the option of overriding the `share_process_namespace` setting?

Reply:
Moved in c58a21c